File: proxy/tests/script_engine/trailer_clear.js
/// <reference path="../../script_libs/js/index.d.ts" />
/** @type {Extension} */
const trailer_clear = {
  request(flow) {
    flow.request.trailers.clear();
  },
  response(flow) {
    flow.response.trailers.clear();
  }
}
globalThis.extensions = [trailer_clear];

File: proxy/tests/script_engine/trailer_clear.lua
pcall(require, "../../script_libs/lua/roxy.lua")
---@type Extension
local trailer_clear = {
	request = function(flow)
		flow.request.trailers:clear()
	end,
	response = function(flow)
		flow.response.trailers:clear()
	end,
}
Extensions = { trailer_clear }
